ckanext/ckanext-apply_permissions_for_service/ckanext/apply_permissions_for_service/views.py

```diff
@@ -136,13 +140,35 @@ def settings_get(context, subsystem_id, errors={}, values=None):
 
 def settings_post(context, subsystem_id):
     form = toolkit.request.form
+
     data_dict = {
-            'subsystem_id': subsystem_id,
-            'delivery_method': form.get('deliveryMethod'),
-            'email': form.get('email'),
-            'api': form.get('api'),
-            'web': form.get('web')
-            }
+        'subsystem_id': subsystem_id,
+        'delivery_method': form.get('deliveryMethod'),
+        'email': form.get('email'),
+        'api': form.get('api'),
+        'file': form.get('file'),
+        'file_url': form.get('file_url'),
+        'clear_upload': form.get('clear_upload'),
+        'web': form.get('web'),
+    }
+
+    # TODO: Fix uploading, I couldn't get it working.
```

Comment:
This needs to be checked. I tried all that I could figure out, but only thing about uploading that I managed to make working was folder creation.

Reply:
Will be handled in LIKA-326